cmd/agent/subcommands/integrations/command.go

```diff
@@ -59,10 +59,10 @@ except PackageNotFoundError:
 
 var (
 	datadogPkgNameRe      = regexp.MustCompile("datadog-.*")
-	yamlFileNameRe        = regexp.MustCompile("[\\w_]+\\.yaml.*")
-	wheelPackageNameRe    = regexp.MustCompile("Name: (\\S+)")                                                                                   // e.g. Name: datadog-postgres
-	versionSpecifiersRe   = regexp.MustCompile("([><=!]{1,2})([0-9.]*)")                                                                         // Matches version specifiers defined in https://packaging.python.org/specifications/core-metadata/#requires-dist-multiple-use
-	pep440VersionStringRe = regexp.MustCompile("^(?P<release>\\d+\\.\\d+\\.\\d+)(?:(?P<preReleaseType>[a-zA-Z]+)(?P<preReleaseNumber>\\d+)?)?$") // e.g. 1.3.4b1, simplified form of: https://www.python.org/dev/peps/pep-0440
+	yamlFileNameRe        = regexp.MustCompile(`[\w_]+\.yaml.*`)
+	wheelPackageNameRe    = regexp.MustCompile(`Name: (\S+)`)                                                                              // e.g. Name: datadog-postgres
+	versionSpecifiersRe   = regexp.MustCompile("([><=!]{1,2})([0-9.]*)")                                                                   // Matches version specifiers defined in https://packaging.python.org/specifications/core-metadata/#requires-dist-multiple-use
+	pep440VersionStringRe = regexp.MustCompile(`^(?P<release>\d+\.\d+\.\d+)(?:(?P<preReleaseType>[a-zA-Z]+)(?P<preReleaseNumber>\d+)?)?$`) // e.g. 1.3.4b1, simplified form of: https://www.python.org/dev/peps/pep-0440
```

Comment:
@pgimalac Were these fixed automatically or by hand? I'm trying to figure out how much to look into this file's changes.

Reply:
Unfortunately they were very much fixed by hand (there is no way to fix with the binary, I read an issue where they said they started working on enabling fixing from the editor through `gopls` but I was unable to do that)

The only two lints which are non-trivial to check are
- `S1007: should use raw string (...) with regexp.MustCompile to avoid having to escape twice`: each character escape should be removed (ie `\x` should become `x`, including if `x` is `\` itself)
- `S1038: should use fmt.Fprintf instead of fmt.Fprintln(fmt.Sprintf(...))`: a newline should be inserted at the end of the message